src/main/java/data/dukedata.txt

```diff
@@ -0,0 +1,3 @@
+1.[T][X] test2              
+2.[T][X] foru    
+3.[D][X] forum enjoyer (by: Tuesday) 
```

Comment:
You seem to have 2 data text file and folder. Just a reminder to double check you file path and delete the redundant folder